network/p2p/p2pnode/libp2pNode.go

```diff
@@ -83,6 +83,11 @@ func NewNode(
 }
 
 // Stop terminates the libp2p node.
+// The following benign errors are expected during normal operations from libP2P:
+// 	- node fails to unsubscribe from all topics
+// 	- p2p host fails to close
+// 	- peer store fails to close
+// All errors returned from this function can be considered benign.
```

Comment:
Can we clarify what we mean by `benign` error? Are we just expected to ignore errors or do we need to check the type of error it is?

Reply:
Per our [CodingConventions](https://github.com/onflow/flow-go/blob/master/CodingConventions.md#best-practice-guidelines) see the section describing a benign error and example godoc comments. Whenever we modify files with un-documented methods we should update the godoc for those.

Reply:
Thanks for the clarification. Can we add a link to the [Coding Convention](https://github.com/onflow/flow-go/blob/master/CodingConventions.md#best-practice-guidelines) in the code so it's more clear for someone unfamiliar with our coding conventions?